pkg/endpoint/policy.go

```diff
@@ -619,14 +621,14 @@ func (e *Endpoint) SetIdentity(owner Owner, id *policy.Identity) {
 
 // l3PolicyValidation prevents code generation failures due to the number of
 // CIDR matches
-func (e *Endpoint) l3PolicyValidation() error {
```

Comment:
We have the `sanitize` methods for all the policies, can you move this as a method of `policy.L3Policy`?

Reply:
Fixed